app.js

```diff
@@ -63,6 +71,18 @@
       if ( this.$('.check.comment').is(':checked') ) {
         results = this.filterCommentResults(results);
       }
+      if ( this.$('.check.created').is(':checked') ) {
+        results = this.filterByCreatedDate(results);
+      }
+      if ( this.$('.check.updated').is(':checked') ) {
+        results = this.filterByUpdatedDate(results);
+      }
+
+      // Remove times from dates
+      _.each(results, function(macro) {
```

Comment:
Why did you change all the forEach methods with the underscore equivalent in this PR?